scripts/prepare-spanner-unit-tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
I attempted a branch with the entrypoint alternative I suggested, it [seems to work](https://app.circleci.com/pipelines/github/mozilla-services/syncstorage-rs?branch=test%2Fspanner-db-tests-ci-STOR-61-entrypoint-pjenvey) and negates the need for separate scripts:

https://github.com/mozilla-services/syncstorage-rs/compare/test/spanner-db-tests-ci-STOR-61...test/spanner-db-tests-ci-STOR-61-entrypoint-pjenvey

Want to merge this in and kill this file?